src/config/logger.ts

```diff
@@ -20,7 +20,7 @@ type CustomLoggerParams = {
     action: string
     [other: string]: any
   }
-  error?: Error
+  error?: unknown
```

Comment:
could you explain why this is necessary?

Reply:
There's no real need to cast this as an Error, so we cast it to a more correct `unknown`. This allows us to not have to cast errors `as Error` when we do not need to unwrap it.